src/containers/LikeDislike/index.js

```diff
@@ -0,0 +1,31 @@
+// import React, { Component } from 'react';
```

Comment:
Hey @imireallan,
How come this entire file has been commented out?

Reply:
@SrMoffat this is because the pr as at now only shows the functionality of reducers and actions the commented file is for the presentational component which am currently doing